docs/team.rst

```diff
@@ -79,9 +79,12 @@ Roles
 Members
 ```````
 
-* `Mike Jang <https://twitter.com/TheMikeJang>`__, Senior Staff Technical Writer, ForgeRock *
-* `Margaret Eker <https://twitter.com/meker>`_
-* `Rose Williams <https://twitter.com/ZelWms>`_
+* `Rose Williams <https://twitter.com/ZelWms>`_ *
+* `Margaret Eker <https://twitter.com/meker>`_ *
```

Comment:
Is Margaret still involved? I thought she stepped down.

Reply:
Margaret ping'd me after the conference, saying that she felt re-energized, and was ready to take more leadership. I figure between Margaret and Rose, I will be able to step back!

I've invited you to our meta-meetup-orgs Trello / Slack.

Reply:
And with the innovations I've seen from Austin in the past year, I think she's well qualified to spread the word on -- (should I call it?) "non-traditional" meetup formats, cc @rosewms